v2/azure-arm.yaml

```diff
@@ -728,12 +728,6 @@ objectModelConfiguration:
       ManagedDiskParameters:
         Id:
           $armReference: true
-      OSProfile:
-        AdminPassword:
-          $isSecret: true
-      VirtualMachineScaleSetOSProfile:
-        AdminPassword:
-          $isSecret: true
```

Comment:
Do these properties exist? If so, they should be treated as secrets.

Reply:
No, they don't exist. These are the redundant ones.